python/runtime/feature/compile.py

```diff
@@ -0,0 +1,164 @@
+# Copyright 2020 The SQLFlow Authors. All rights reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np
+import six
+from runtime.feature.column import (BucketColumn, CategoryHashColumn,
+                                    CategoryIDColumn, CrossColumn,
+                                    EmbeddingColumn, IndicatorColumn,
+                                    NumericColumn, SeqCategoryIDColumn)
+from runtime.feature.field_desc import DataType
+from runtime.model import EstimatorType
+
+__all__ = [
+    'compile_ir_feature_columns',
+]
+
+
+def to_package_dtype(dtype, package):
+    """
+    Convert dtype to the data type accepted by package.
+
+    Args:
+        dtype (DataType): one of INT, FLOAT and STRING.
+        package (module): the Python package.
+
+    Returns:
+        The data type accepted by the package.
+    """
+    if dtype == DataType.INT:
+        return package.dtypes.int64
+
+    if dtype == DataType.FLOAT:
+        return package.dtypes.float32
+
+    if dtype == DataType.STRING:
+        return package.dtypes.string
+
+    raise ValueError("unsupported data type {}".format(dtype))
+
+
+def compile_feature_column(fc, model_type, package):
+    """
+    Compile a IR FeatureColumn object to a runtime feature column object.
+
+    Args:
+        fc (FeatureColumn): the IR FeatureColumn object.
```

Comment:
How about marking fc specifically to `fc_ir`

Reply:
Done. Renamed to `ir_fc`.